### PR TITLE
adding ring hash lb and tweaks to LB

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

**Comment by rshriram** [comment]
cc @pacevedom FYI. This includes ringhash LB. But this is in v1alpha2 only.

File: routing/v1alpha2/destination_rule.proto [hunk 1]
```diff
@@ -165,6 +149,92 @@ message Subset {
   TrafficPolicy traffic_policy = 3;
 }
 
+// Load balancing policies to apply for a specific destination. See Envoy's
+// [load balancing](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing.html)
```
Comment (ZackButcher) [inline routing/v1alpha2/destination_rule.proto]: See Envoy's load balancing __documentation__ for more details.

File: routing/v1alpha2/destination_rule.proto [hunk 2]
```diff
@@ -165,6 +149,92 @@ message Subset {
...
+//       name: ratings
+//       trafficPolicy:
+//         loadBalancer:
+//           consistentHash:
+//             http_header: Cookie
+//
+message LoadBalancerSettings {
+  // Standard load balancing algorithms that require no tuning.
+  enum SimpleLB {
+    // round robin policy. Default
```
Comment (ZackButcher) [inline routing/v1alpha2/destination_rule.proto]: // Round Robin policy. Default.

**Pull request closed without merging**